FreeRTOS-Plus/Test/CBMC/proofs/Socket/vSocketBind/ALLOW_SOCKET_SEND_WITHOUT_BIND/vSocketBind_harness.c

```diff
@@ -0,0 +1,69 @@
+/* Standard includes. */
+#include <stdint.h>
+#include <stdio.h>
+
+/* FreeRTOS includes. */
+#include "FreeRTOS.h"
+#include "list.h"
+
+/* FreeRTOS+TCP includes. */
+#include "FreeRTOS_IP.h"
+#include "FreeRTOS_IP_Private.h"
+#include "FreeRTOS_Sockets.h"
+
+
+void *safeMalloc(size_t xWantedSize) {
+        return nondet_bool() ? malloc(xWantedSize) : NULL;
+}
+
+uint16_t prvGetPrivatePortNumber( BaseType_t xProtocol )
+{
+	uint16_t usResult;
+	return usResult;
+}
+
+BaseType_t xIPIsNetworkTaskReady( void )
+{
+	/* Return true saying that the task is ready. */
+	return pdTRUE;
+}
+
+
+/* Random number generator provided by the application. In our case, CBMC provides
+ * an indeterministic value. */
+BaseType_t xApplicationGetRandomNumber( uint32_t *pulNumber )
+{
```

Comment:
Test for nonnull?

Reply:
Added, thanks!

Reply:
Let's merge this, but checkout https://github.com/aws/amazon-freertos/blob/master/tools/cbmc/proofs/Queue/QueueGenericCreate/Configurations.json for an example of how to use one harness and use to test multiple configuration.